torch/_meta_registrations.py

```diff
@@ -6379,6 +6379,165 @@ def ceil_div(a, b):
     return torch.empty(self.size(0), mat2.size(1), dtype=_out_dtype, device=self.device)
 
 
+@register_meta([aten._scaled_grouped_mm.default])
```

Comment:
Can you avoid moving `meta_scaled_grouped_mm` around in this PR?  It makes it hard to see what all changed.  (If it's being moved for a good reason, can you explain what that is?  It's hard for me to tell from the diff)

Reply:
Moved back.  (I've added it in my PR before it get added into the main, and I though it may make sense to put it right after meta registration for `_scaled_mm`.)

Reply:
I put it next to grouped_mm which also makes sense, scaled_grouped_mm has 2 attributes and it's unclear which one is more important ;-)